src/lib/Scenes/Artwork/Components/CommercialInformation.tsx

```diff
@@ -76,27 +76,27 @@ export class CommercialInformation extends React.Component<CommercialInformation
     const { editionSetID } = this.state
     const { isAcquireable, isOfferable, isInquireable, isInAuction, sale, isForSale } = artwork
 
-    const showAuctionTimerAndButton = isInAuction && sale && !sale.isClosed && isForSale
-    const shouldRenderButtons = isAcquireable || isOfferable || isInquireable || showAuctionTimerAndButton
-    const consignableArtistsCount = artwork.artists.filter(artist => artist.isConsignable).length
+    const isBiddableInAuction = isInAuction && sale && !sale.isClosed && isForSale
```

Comment:
I renamed these slightly since I was using them for a slightly different purpose and this felt clearer. Open to suggestions, though!